build.gradle.kts

```diff
@@ -34,7 +34,6 @@ buildscript {
     val jvmToolChainVersion by extra(17)
     val coreKtxVersion by extra("1.13.1")
     val sqliteAndroidVersion by extra("3.42.0")
-    val jswordVersion by extra("2.4.14")
```

Comment:
submodule requires CI stuff to be working also, will consider if I fix them or drop submodule stuff from here (of course help appreciated if possible).

Reply:
I'll have a look next week to see if I can get the pipelines running correctly.

Reply:
could also be something chatgpt will give us with a simple prompt